config/application.py

```diff
@@ -15,6 +15,7 @@
     'masonite.providers.StartResponseProvider.StartResponseProvider',
     'masonite.providers.SassProvider.SassProvider',
     'masonite.providers.WhitenoiseProvider.WhitenoiseProvider',
+    'masonite.providers.CacheProvider.CacheProvider',
```

Comment:
Because wsgi is False, we can move this all the way to the bottom of the list. Service providers should be additive and shouldn't have to be injected in random places.